### ISSUE TITLE
Shiny Mew and Hidden Power Issues

### ISSUE THREAD

**Issue by ReZeroEmiria** [architdate/PKHeX-Auto-Legality-Mod#12]
Greetings. I managed to use RNGReporter to alter a mew generated by this mod to be shiny. (Faraway Island Mew) This is possible however the program will not allow me to do so just with the modded import. Also I tried to generate an older generation pokemon shiny and with a specific HP type (Hidden Power Fire in this case.) and it won't give it the correct Hidden Power.

Also just tried this to mess around with the capabilities:

Mesprit @ Leftovers
Shiny: Yes
EVs: 4 HP / 252 SpA / 252 Spe
Timid Nature
- Stealth Rock
- Hidden Power [Fighting]
- Thunderbolt
- U-turn

And it gave me an error, a non-shiny one works though.

**Comment by Pierrounator** [comment]
The Mesprit with Hidden Power [Fire] works fine for me... Same with Fighting 🤔

**Comment by architdate** [comment]
@ReZeroEmiria did you try mew with the latest commit? Or with the latest release? I believe I fixed shiny mew in a previous issue report

**Comment by architdate** [comment]
Mespirit issue cant be replicated 
![image](https://user-images.githubusercontent.com/17220860/30223549-e8b9a906-94fd-11e7-928a-4f15534a3257.png)
![image](https://user-images.githubusercontent.com/17220860/30223558-f473c9a2-94fd-11e7-996b-ee3ba01f880d.png)

**Comment by ReZeroEmiria** [comment]
@architdate All the things I've reported I've done with the latest release you have on here (8 Days ago). I haven't tried any of this by compiling my own with the latest 9-5-17 PKHex. (The Day I discovered this tool and decided to use it I was having a major headache due to some code revisions and research for a project that I didn't want to compile anything haha.)

I redownloaded and retried and the Mesprit works. As I explained, everything I've done is with latest release not commit.
